### ISSUE TITLE
AttributeError: 'Call' object has no attribute 'starargs'

### ISSUE THREAD

**Issue by flyhunter** [lmfit/lmfit-py#281]
I'm using expressions for parameters and I'm getting an error. My code is:

```python
x = [1723, 1773, 1823, 1523, 1773, 1033.03078, 1042.98077, 1047.90937, 1053.95899, 1057.94906, 1063.13788, 1075.74218, 1086.03102]
y = [0.79934, -0.31876, -0.46852, 0.05, -0.21, 11.1708, 10.31844, 9.73069, 9.21319, 9.12457, 9.05243, 8.66407, 8.29664]

def VFT(T, ninf=-3, A=5e3, T0=800):
 return ninf + A/(T-T0)

vftModel = Model(VFT)
vftModel.set_param_hint('D', vary=False, expr=r'A*log(10)/T0')
result = vftModel.fit(y, T=x)
print(result.fit_report())
```

The error reads:

     - Adding parameter for hint "D"
     - Adding parameter "ninf"
     - Adding parameter "A"
     - Adding parameter "T0"
     - Adding parameter "D"
    AttributeError
       <_ast.Module object at 0x7efded66e6d8>
          ^^^
    'Call' object has no attribute 'starargs'

    AttributeError:  in expr='<_ast.Module object at 0x7efded66e6d8>'

However, everything runs smoothly when I run the above code without using "log" in the expression:

```python
vftModel.set_param_hint('D', vary=False, expr=r'A/T0')
```

I'm using Python3.5 in a fully updated Arch Linux (as of 5 Dec 2015) with the latest lmfit (0.9.2).

**Comment by newville** [comment]
@flyhunter  thanks!  This looks like a good hint:

https://github.com/alecthomas/importmagic/commit/4f26647461165a6ffa163f4d745bf52d71bb1027

**Comment by newville** [comment]
Should be addressed with #282
